reference/open_pbr_surface.mtlx

```diff
@@ -580,10 +589,14 @@
     <luminance name="opacity_luminance" type="color3">
       <input name="in" type="color3" interfacename="geometry_opacity" />
```

Comment:
Note that in the spec, `geometry_opacity` is defined to be a float. So it's unclear why it was set to be a color in the MaterialX inputs?

Reply:
Good catch, and that's very likely a leftover from the original definition of `opacity` in Standard Surface.  I can make the change here as part of our upgrade to MaterialX 1.39.